crates/host/src/policy.rs

```diff
@@ -148,58 +172,30 @@ fn is_expired(expires: u64) -> bool {
         > expires
 }
 
-impl From<jwt::Claims<jwt::Actor>> for RequestSource {
-    fn from(claims: jwt::Claims<jwt::Actor>) -> Self {
-        RequestSource {
-            public_key: Some(claims.subject),
-            contract_id: None,
-            link_name: None,
-            capabilities: claims.metadata.and_then(|m| m.caps).unwrap_or_default(),
-            issuer: Some(claims.issuer),
-            issued_on: Some(claims.issued_at.to_string()),
+impl From<&jwt::Claims<jwt::Actor>> for PolicyClaims {
+    fn from(claims: &jwt::Claims<jwt::Actor>) -> Self {
+        PolicyClaims {
+            public_key: claims.subject.to_string(),
+            issuer: claims.issuer.to_string(),
+            issued_at: claims.issued_at.to_string(),
             expires_at: claims.expires,
             expired: claims.expires.map(is_expired).unwrap_or_default(),
         }
     }
 }
 
-impl From<jwt::Claims<jwt::CapabilityProvider>> for RequestSource {
-    fn from(claims: jwt::Claims<jwt::CapabilityProvider>) -> Self {
-        RequestSource {
-            public_key: Some(claims.subject),
-            contract_id: claims.metadata.map(|m| m.capid),
-            link_name: None, // Unfortunately, since claims don't include a link name, we can't populate this
-            capabilities: vec![],
-            issuer: Some(claims.issuer),
-            issued_on: Some(claims.issued_at.to_string()),
+impl From<&jwt::Claims<jwt::CapabilityProvider>> for PolicyClaims {
```

Comment:
Nit: Probably still want the owned version of these too

Reply:
I figured I'd implement this once we actually need it instead of having the impl without the need, if that makes sense. Will add if it helps address remaining PR comments

Reply:
Nah you're good